src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1508,6 +1514,23 @@ namespace ReportSizingManager {
 						}
 					}
 					bCheckForZero = false;
+				} else if ( SizingType == DesiccantRegCoilDesAirInletTempSizing ) {
+					if ( DesicDehum( DataDesicDehumNum ).RegenInletIsOutsideAirNode ) {
+						AutosizeDes = FinalSysSizing( CurSysNum ).HeatOutTemp;
+					} else {
+						AutosizeDes = FinalSysSizing( CurSysNum ).HeatRetTemp;
+					}
+					bCheckForZero = false;
+				} else if ( SizingType == DesiccantRegCoilDesAirOutletTempSizing ) {
+					AutosizeDes = DesicDehum( DataDesicDehumNum ).RegenSetPointTemp;
+					bCheckForZero = false;
```

Comment:
I'd rather see a AutosizeDes = DataDesInletAirTemp used here instead of a specific structure variable. And then if that is the  case, a very generic SizingType name could be used (i.e., SizingType == DesiccantRegCoilDesAirOutletTempSizing  is very specific).